common/scripted_effects/wc_scheme_effects.txt

```diff
@@ -0,0 +1,238 @@
+﻿#Sets up the Outcome Roll values
+stasis_trap_outcome_roll_setup_effect = {
+	#Discovery Roll setup
+	save_scope_value_as = {
+		name = discovery_chance
+		value = {
+			value = 100
+			subtract = scope:scheme.scheme_secrecy
+		}
+	}
+}
+
+#Roll to check whether a Stasis Trap Scheme is a success or a failure, and whether the owner is discovered
+stasis_trap_outcome_roll_effect = {
+	#SUCCESS ROLL
+	random = {
+		chance = scope:scheme.scheme_success_chance
+		custom_tooltip = stasis_trap_successful_roll_tt
+		save_scope_value_as = {
+			name = scheme_successful
+			value = yes
+		}
+	}
+	### End Success roll
+	#DISCOVERY ROLL
+	random = {
+		chance = scope:discovery_chance
+		custom_tooltip = stasis_trap_become_discovered_roll_tt
+		save_scope_value_as = {
+			name = scheme_discovered
+			value = yes
+		}
+	}
+	### End Discovery roll
+
+	hidden_effect = {
+		#FIRE CORRECT ON ACTIONS
+		if = {
+			limit = {
+				exists = scope:scheme_successful
+			}
+			trigger_event = {
+				on_action = stasis_trap_succeeded
+			}
+		}
+		else = {
+			trigger_event = {
+				on_action = stasis_trap_failed
+			}
+		}
+	}
+}
+
+
+#Used in the immediate of the Scheme Owner's success event
+stasis_trap_success_effect = {
+	
+	scope:target = {
+		save_scope_value_as = {
+			name = scheme_discovered
+			value = $DISCOVERED$
+		}
+
+		trigger_event = stasis_trap_outcome.5001
+	}
+
```

Comment:
So the murder scheme gives stress reduction and I think it's nice.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/117540977-c5de7800-b022-11eb-9bd4-bfb8ef2fd2c5.png)

</details>